go-controller/pkg/ovn/gateway.go

```diff
@@ -388,10 +388,11 @@ func (gw *GatewayManager) GatewayInit(
 		}
 		if gw.netInfo.IsPrimaryNetwork() && gw.netInfo.TopologyType() == types.Layer2Topology {
 			logicalRouterPort.Ipv6RaConfigs = map[string]string{
-				"address_mode":  "dhcpv6_stateful",
-				"send_periodic": "true",
-				"max_interval":  "900", // 15 minutes
-				"min_interval":  "300", // 5 minutes
+				"address_mode":      "dhcpv6_stateful",
+				"send_periodic":     "true",
+				"max_interval":      "900", // 15 minutes
+				"min_interval":      "300", // 5 minutes
+				"router_preference": "LOW",
```

Comment:
nit; please add comment for why "LOW" is being set
this is for when the pod ends up with a LLA of join subnet right?
maybe if we go with what we discussed yesterday around using podsubnet IPs on GR for it will make things easier cause then the LLA sent back will be the same